--- schemas.py.old	2022-01-26 09:20:41.000000000 -0500
+++ schemas.py	2022-01-26 09:23:19.000000000 -0500
@@ -88,6 +88,9 @@
         AssociatedDnsProviderSchema, missing=None, allow_none=True, required=False
     )
 
+    wildcard = fields.Boolean(default=False)
+    gsproduct = fields.String()
+
     csr = fields.String(allow_none=True, validate=validators.csr)
 
     key_type = fields.String(
@@ -297,6 +300,8 @@
     issuer = fields.String()
     name = fields.String()
     dns_provider_id = fields.Integer(required=False, allow_none=True)
+    wildcard = fields.Boolean(default=False)
+    gsproduct = fields.String()
     date_created = ArrowDateTime()
     resolved = fields.Boolean(required=False, allow_none=True)
     resolved_cert_id = fields.Integer(required=False, allow_none=True)
